test/factories/contacts.rb

```diff
@@ -2,11 +2,16 @@
   factory :contact do
     sequence :first_name { |n| "first name #{n}" }
     sequence :last_name { |n| "last name #{n}" }
-    street 'Strassenstr. 223'
     extended 'asdfadsf'
-    city 'Zürich'
+    street 'Strassenstr. 223'
+    city 'Rüdisüligen'
```

Comment:
If you ever need random data, the [ffaker gem](https://github.com/ffaker/ffaker) is great for this.

Reply:
I've lost my path last week, with trying to user factory_girl for seeds with faker (*trashed a bit of my unprecious time*)... so I have a faker fake history ... 😉 
But yeah, right.